MaxText/layers/llama2.py

```diff
@@ -42,15 +44,88 @@
 # -----------------------------------------
 
 
-class LlamaDecoderLayer(nn.Module):
+class LlamaDecoderLayer(nnx.Module):
   """Transformer decoder layer that attends to the encoder."""
 
-  config: Config
-  mesh: Mesh
-  model_mode: str
-  quant: Optional[Quant] = None
+  def __init__(
+      self,
+      config: Config,
+      model_mode: str,
+      mesh: Mesh,
+      quant: Optional[Quant] = None,
+      rngs: Optional[nnx.Rngs] = None,
```

Comment:
Rngs should be required
```suggestion
      *,
      rngs: nnx.Rngs = None,
```